src/routes/PostDetail/Comments/CommentForm/CommentForm.js

```diff
@@ -51,11 +52,12 @@ class CommentForm extends Component {
       createComment,
       sendIsTyping,
       attachments,
-      clearAttachments
+      clearAttachments,
+      t
     } = this.props
 
     if (this.editor?.current && this.editor.current.isEmpty()) {
-      // Do nothing and stop event propagation
+      window.alert(t('You need to include text to post a comment'))
```

Comment:
you know, it seems like we should allow comments that are just images... but maybe thats another ticket. I could also see this one being kind of annoying. I dont think anyone would expect that hitting enter with an empty comment would do anything, but now it pops up a window alert you have to close. maybe we only do an alert if there's an image but no text? or maybe we just change it so that if there's an image but no text we set the comment text to a blank space and let it be created?

Reply:
Ok, changed this to allow comment posts with just images.